new_game.py

```diff
@@ -0,0 +1,139 @@
+# Python 2
+from PyDictionary import PyDictionary
+import random, time, os
+
+class Data(object):
+
+    # Builds the word database
+    def collect_words(word_file="gre_word_file.txt"):
+        wordlist = open(word_file).read().splitlines()
+        return wordlist
+
+    def __init__(self, words=collect_words()):
+        self.words = words
+        self.lookup = PyDictionary()
+
+    def display_words(self):
+        for word in self.words:
+            print word
+            
+    # Chooses a random word from the wordlist
+    def answer(self):
+        answer_idx = random.randint(0, len(self.words) - 1)
```

Comment:
It should always have a huge list, but yea if I use some new list of words, it could break. What do you suggest?
I want to eventually be able to use any list within a directory, to have an options menu to do that.